packages/chart/src/components/LineChart/components/LineChart.Circle.tsx

```diff
@@ -178,19 +179,24 @@ const LineChartCircle = (props: LineChartCircleProps) => {
     }
   }
   if (mode === 'ISOLATED_POINTS') {
+    const findDataIndex = () => {
+      const indexNumber = data.findIndex(item => item[config.xAxis.dataKey] === pointData[config.xAxis.dataKey])
+      return indexNumber
+    }
     const drawIsolatedPoints = (currentIndex, seriesKey) => {
-      const currentPoint = data[currentIndex]
-      const previousPoint = data[currentIndex - 1] || {}
-      const nextPoint = data[currentIndex + 1] || {}
+      const workingIndex = pointData ? findDataIndex() : currentIndex
```

Comment:
don't defined workingIndex just pass in the working index. you don't need to change drawIsolatedPoints function at all.